ocaml/idl/datamodel.ml

```diff
@@ -8638,6 +8649,47 @@ let vgpu_type =
 		]
 	()
 
+(** PVS_farm
+ * http://www.uk.xensource.com/~robho/pvs-direct/#external-interfaces-xenapi-pvs_farm-new-class
+ *)
+
+
+let pvs_farm =
+  let lifecycle = [Published, rel_dundee_plus, ""] in
+  let refset x  = Set(Ref x) in
+    create_obj
+    ~name: _pvs_farm
+    ~descr:"machines serving blocks of data for provisioning VMs"
+    ~doccomments:[]
+    ~gen_constructor_destructor:false
+    ~gen_events:true
+    ~in_db:true
+    ~lifecycle:lifecycle
```

Comment:
Here, and below, you could just write `~lifecycle` rather than `~lifecycle:lifecycle`.